packages/build-tools/src/utils/expoUpdates.ts

```diff
@@ -114,19 +114,12 @@ export async function configureClassicExpoUpdatesAsync(ctx: BuildContext<Job>):
   if (ctx.job.releaseChannel) {
     await setClassicReleaseChannelNativelyAsync(ctx);
   } else {
-    /**
-     * If releaseChannel is not defined:
-     *  1. Try to infer it from the native value.
-     *  2. If it is not set, fallback to 'default'.
-     */
     const releaseChannel = await getNativelyDefinedClassicReleaseChannelAsync(ctx);
     if (releaseChannel) {
       ctx.logger.info(
         `Using the release channel pre-configured in native project (${releaseChannel})`
       );
       ctx.logger.warn('Please add the "releaseChannel" field to your build profile (eas.json)');
-    } else {
-      ctx.logger.info(`Using default release channel for 'expo-updates' (default)`);
```

Comment:
For a long time, we will still build projects with an old version of expo-updates package. I suggest instead fo removing this log, to change it to sth like "No channel configured, continuing with default behavior", or maybe checking version of expo-updates package, but that might be an overkill.